driver-sync/src/main/com/mongodb/client/vault/ClientEncryption.java

```diff
@@ -190,6 +195,32 @@ public interface ClientEncryption extends Closeable {
      */
     RewrapManyDataKeyResult rewrapManyDataKey(Bson filter, RewrapManyDataKeyOptions options);
 
+    /**
+     * {@linkplain MongoDatabase#createCollection(String, CreateCollectionOptions) Create} a new collection with encrypted fields,
+     * automatically {@linkplain #createDataKey(String, DataKeyOptions) creating}
+     * new data encryption keys when needed based on the configured
+     * {@link CreateCollectionOptions#getEncryptedFields() encryptedFields}, which must be specified.
+     * This method does not modify the configured {@code encryptedFields} when creating new data keys,
+     * instead it creates a new configuration if needed.
+     *
+     * @param database The database to use for creating the collection.
+     * @param collectionName The name for the collection to create.
+     * @param createCollectionOptions Options for creating the collection.
+     * @param createEncryptedCollectionParams Auxiliary parameters for creating an encrypted collection.
+     * @return The (potentially updated) {@code encryptedFields} configuration that was used to create the collection.
+     * A user may use this document to configure {@link AutoEncryptionSettings#getEncryptedFieldsMap()}.
+     * @throws MongoUpdatedEncryptedFieldsException If an exception happens after creating at least one data key.
+     * This exception makes the updated {@code encryptedFields}
+     * {@linkplain MongoUpdatedEncryptedFieldsException#getEncryptedFields() available} to the caller.
+     *
+     * @since 4.9
+     * @mongodb.server.release 6.0
+     * @mongodb.driver.manual reference/command/create Create Command
+     */
+    @Beta(Beta.Reason.SERVER)
+    BsonDocument createEncryptedCollection(MongoDatabase database, String collectionName, CreateCollectionOptions createCollectionOptions,
```

Comment:
`MongoUpdatedEncryptedFieldsException` is unchecked because `MongoException` is unchecked, and `MongoUpdatedEncryptedFieldsException` is of the `MongoException` type.

If you are talking about `throws`, then the JLS says: [_"It is permitted but not required to mention unchecked exception classes ([§11.1.1](https://docs.oracle.com/javase/specs/jls/se19/html/jls-11.html#jls-11.1.1)) in a throws clause."_](https://docs.oracle.com/javase/specs/jls/se19/html/jls-8.html#jls-8.4.6:~:text=It%20is%20permitted%20but%20not%20required%20to%20mention%20unchecked%20exception%20classes%20(%C2%A711.1.1)%20in%20a%20throws%20clause.) It appears that we use it in the public API extremely rarely, but we do: `DnsClient.getResourceRecordData`.

Since this appear to cause confusion, I'll remove the `throws` clause.